.gitignore

```diff
@@ -0,0 +1,10 @@
+week5/5.1/.dist/
+week5/5.2/.dist/
+week5/5.3/.dist/
+week5/5.4/.dist/
+week6/6.1/.dist/
+week6/6.2/.dist/
+week6/6.3/.dist/
+week6/6.4/.dist/
+week6/6.5/.dist/
+.dist/
```

Comment:
Also, I think you would like to search online for generic gitignores - python and pycharm, and add them to your main branch, then merge your main to here